web/src/utils/Context.ts

```diff
@@ -55,7 +57,8 @@ const FilterListContextDefaults: FilterListState = {
 function ContextMerger<T extends {}>(
   key: string,
   defaults: T,
-  ctxState: StateWithValue<T>
+  ctxState: StateWithValue<T>,
+  incognitoKey?: string
```

Comment:
This extra thing shouldn't be needed I think? We just keep the single SettingsKey so there's no separate handling, all done via a single context for settings.

Reply:
now its good I think, no separate key for local storage now